great_expectations/data_context/store/database_store_backend.py

```diff
@@ -138,7 +138,7 @@ def _convert_engine_and_key_to_url(self, key):
         full_url = str(self.engine.url)
         engine_name = full_url.split("://")[0]
         db_name = full_url.split("/")[-1]
-        return engine_name + "://" + db_name + "/" + str(key)
+        return engine_name + "://" + db_name + "/" + str(key[0])
```

Comment:
I realized that this key comes in as a tuple, rather than a string. It's mostly a visual change, and (fortunately) doesn't have any functional consequences. I adjusted the unittests accordingly

Reply:
Interesting that this wasn't causing issues elsewhere. Good find!